drivers/SmartThings/matter-appliance/src/init.lua

```diff
@@ -33,14 +34,23 @@ if version.api < 10 then
   clusters.TemperatureControl = require "TemperatureControl"
 end
 
+if version.api < 11 then
+  clusters.MicrowaveOvenControl = require "MicrowaveOvenControl"
+  clusters.MicrowaveOvenMode = require "MicrowaveOvenMode"
+  clusters.OvenMode = require "OvenMode"
```

Comment:
OvenMode is not present in 54 lua libs so this should be moved outside of the if statement